swahili/marshmallow.md

```diff
@@ -0,0 +1,3 @@
+Asubuhi moja, nimeamka ndani ya chumba cha ajabu kikiwa na marshmallow kwenye kuta zake.
```

Comment:
You can fix this by adding a link in `language.md` that says "If you speak swahili, click here" (in swahili, though).